### DISCUSSION TITLE
Modeling of future scenarios Q&A

### DISCUSSION THREAD

**Discussion by gavinny** [NASA-GISS/freshwater-forcing-workshop#12] [Talks Q&A]
Use this thread to ask questions related to the modeling of future scenarios talks in section 3.

**Comment by dfelikson** [comment]
The question of using ISMIP output as freshwater forcing in scenario projections within ESMs came up during the Climate Modeling Results and Implementations session (section 2) and I wanted to follow up here. I followed up with Sophie Nowicki and, although modeling calving processes in ISMs is very challenging, there are experiments in ISMIP that prescribed ocean forcing in such a way that all models could implement to simulate future increases in calving flux. The parameterized ocean forcing used in the ISMs is described in [Slater et al. (2020)](https://doi.org/10.5194/tc-14-985-2020) (this is what I had mentioned in the Zoom chat ... note that this is for Greenland and not Antarctica) and the resulting dynamical ice thickness change is shown in [Goelzer et al. (2020)](https://doi.org/10.5194/tc-14-3071-2020) in their Figure 11 (and Figure S9 for all models). 

As a starting point, this can be used to calculate Greenland  calving flux as a forcing for ESMs. The big caveat is that this approach was a very rudimentary way of forcing ISMs and was somewhat overly prescriptive because it specifies the future ice sheet retreat, instead of letting the ISM calculate that itself. Nevertheless, this approach allowed all ISMIP models to participate in the experiments. Again, this might be a good starting point for future anomalous freshwater forcing in ESMs and the ISM ocean forcing protocol is being revisited for ISMIP7.

**Reply by xylar** [reply]
As one of the folks that helped design the ISMIP6 parameterization for Antarctic ice-shelf basal melt in [Jourdain et al. (2020)](https://tc.copernicus.org/articles/14/3111/2020/), I would also call it "rudimentary".  While I think it would be reasonable to use ISMIP6 results to get a sense of the combined freshwater flux from calving and basal melt, I don't think either calving processes or the ice-shelf melt parameterization are sophisticated enough to do a great job at partitioning the relative fraction of melt vs. calving.  At the same time, they and similar standalone ISM simulations may be all we have at this point to partition the melt and calving responses in future projections.

**Reply by crodehacke** [reply]
In my experience of tuning the parameters describing the basal melting in ice shelves for contemporary geometries and existing hydrographic data sets (WOA2018 or ISMIP data set), we can find good combinations as long as we keep the geometry fixed.

However, everything may diverge from the observed state once the model evolves freely. Even if we nearly keep the grounding line and have an acceptable representation of the ice shelf edges, ice shelves might be thinner than observed. Since the ice shelf base determines the susceptibility to the predominant thermal forcing via the pressure-dependent melting, these thinner ice shelves are subject to reduced basal melting rates; luckily, enhanced iceberg calving has kept ice shelves in balance in my simulations. What does it mean?

The participation between melting and calving is not necessarily well represented. I have a suspicion that it goes back to applying not adequate forcing. For instance, hydrographic climatological data and utilized atmospheric boundary conditions from regional atmospheric climate models, such as MAR, RACMO, HIRHAM, and HARMONIE, may not be consistent --- why should they?

However, I'm optimistic that models will improve, more consistent data sets will emerge, and simulated general trends are already helpful.

 **Here I talk about basal melting of floating ice shelves only**

**Reply by nicojourdain** [reply]
For Antarctica, I would consider using the projections of Vio Coulon and Frank Pattyn (see [their presentation](https://drive.google.com/file/d/1C6IqvOfSil-vrWTfSTZ9t1DIiAE1bDp6/view?usp=drive_link) in this workshop). [Their paper](https://doi.org/10.5194/tc-18-653-2024) was published yesterday in TC. The main strength is that it is a large ensemble that explores the uncertainty of multiple model parameters (including for basal and surface mass balances) and then individual members are weighted according to their distance to some observations. It goes from 1950 to 2300 and includes some representation of the isostasy and elevation feedback. A caveat is that it is a single ice sheet model.

**Reply by crodehacke** [reply]
I agree that one model might not be enough. An ensemble of calibrated models or well-trained emulators may lead to a better representation.

**Reply by dfelikson** [reply]
There are also perturbed-parameter ensembles for Greenland (e.g., [Aschwanden et al., 2019](https://doi.org10.1126/sciadv.aav9396); [Choi et al., 2021](https://doi.org10.1038/s43247-021-00092-z)) that do a nice job with sampling uncertainty in ice sheet model parameters but these are also single-model ensembles and, as @nicojourdain and @crodehacke point out, that may not suffice. For multi-model ensembles, and to have stronger connection to CMIP, ISMIP is the best bet for forcing ESMs. This might not be feasible in terms of the timeline (i.e., the ESMs will have to wait for ISMIP7 output, and ISMIP7 has to wait for CMIP7 outputs to run the standalone ISMs), and ISMIP6 can thus be considered as a "rudimentary" starting point.

**Reply by swartn** [reply]
Agreed, it won't be possible to have CMIP7 GCM forcing driving offline ISMs, which then provide forcing back to CMIP7 GCMs in a practical and timely way. Using CMIP6 based results could be done in a few ways I can think of, each with their own challeneges:

1. Run CMIP7 models but with CMIP6 climate forcing + CMIP6 based ISM generated FW forcing. This would be a dedicated MIP activity (i.e. not ScenarioMIP but maybe e.g. SOFIA), and probably a bit complicated since some GCMs will not maintain the ability to use CMIP6 forcing. And it will be a bit dated.
2. Use the CMIP6 forced offline ISM results, and build an emulator, that can then be used to emulate FW fluxes but for CMIP6 forcing. This has its own complications, including the robustness of the emulator, but also, there is again some circular reasoning, because CMIP7 climate would be needed to run the emulator to generate the FW fluxes.
3. Use the CMIP6 forced offline ISM results, and apply these FW forcing rates to the closest CMIP7 scenario. There is nothing circular here, and the timeline is workable, but obviously there will be a slight inconsistency between the climate and FW forcing, which might be deemed acceptable or not.

I'm not sure if there is a better compromise, but I think some compromise has to be made in the context of offline ISM derived forcing.

**Reply by OlgaSergienko** [reply]
@nicojourdain I'd also caution to use the results of any ice-sheet model that employs parameterization of the ice flux through the grounding line (e.g., the Kori-ULB, David Pollard's model) instead of properly resolving stresses there. The differences between parameterized fluxes and properly resolved could be of the orders of magnitude.

**Reply by tamsinedwards** [reply]
We're currently working on the next generation of multi-model, multi-input land ice emulators in FACTS (Kopp et al., 2023, GMD - used for AR6 SL projections), built with PROTECT (and potentially some other) ensembles. In theory this could be adapted to predict any output (not just sea level contribution), which would address some of the questions above. FACTS predicts for any set of GSAT trajectories, which gives a first order scenario consistency.

But it's still a work in progress - I think it would be beyond the scope of needs here, in terms of both timeline of the original work and capacity to repredict for other variables, unless we could get some extra support (which is not impossible).

**Reply by dfelikson** [reply]
@tamsinedwards - if I'm understanding correctly, you're saying that FACTS could be adapted to produce distributions of freshwater flux, calving flux, etc. Am I correct? If so, that would be a very impactful advance and I'm sure those outputs would be widely used by the community. For this immediate effort of forcing ESMs with freshwater flux (liquid and solid) from ISMs, I think the direct outputs from ISMIP are what's needed, rather than a complete probability distribution of these quantities. Am I thinking about that correctly? In my mind, the logical flow is: 

`scenario` -> `ESM (without interactive ice sheet)` -> `standalone ISM` -> `freshwater forcing back to ESM`

This way (theoretically), an ESM can do a finite number of extra runs for each scenario, forced by outputs from ISMIP6 or ISMIP7.

**Reply by tamsinedwards** [reply]
Hi Denis, rather less strong I'm afraid - I'm saying my module (emulandice) in FACTS could, in theory, be developed by me to also predict other things, if I had support to do so. A lot of the work is making the infrastructure, and in theory one could swap one variable for another in the input data and redo everything. Especially if had in exactly the same format as the volume changes (which is not trivial, as any MIPer knows).

But of course emulators need a lot of work to optimise and validate, so every new variable would be an entire new (large-ish) analysis. And, as you say, this wouldn't necessarily be that useful/essential for the forcings themselves - only to provide context, in the form of distributions. Something for a grant proposal, perhaps :)

**Reply by hseroussi** [reply]
We will just have to be careful about the calving flux (ice discharge at the front) since this is not a variable reported by all the models. It is also difficult to evaluate and has a lot of noise, so we would have to be careful about where it comes from and how it is computed. 
If models were perfectly conservative, this could be estimated as the last component of the mass budget, but the models are not always perfectly conservative.

**Comment by Recalculate** [comment]
A question for modellers, from an ignorant observationalist.  Paul Holland's talk on the coupled UKESM + ISM noted that under strong climate forcing SSP585 the Antarctic ice sheet actually grew slightly in mass, due to increased snowfall for only a small net change.  In a non coupled ISM climate model, this extra precipitation would immediately be added back to the ocean as runoff, roughly equal to the modelled melt in the coupled UKESM+ISM.  While vertical/horizontal distribution obviously matters, does this suggest that the current CMIP6 runoff under this scenario are somewhat close to the projected melt flux under (for eg) ISMIP6?    Has anyone looked at this, is UKESM+ISM an outlier, or should we perhaps not expect dramatic changes if we do proscribe this flux (and presumably remove the existing runoff term)?

**Reply by torgemartin** [reply]
I have a related question: We discuss adding a freshwater anomaly (anomaly with respect to the observational estimates of the ice sheet in balance). This will, however, also be an anomaly/perturbation with respect to the individual climate model's mean state runoff from Greenland and Antarctica (which is excess precipitation). How do we deal with precipitation change under global warming? Should this model-specific anomaly rather not be added back into the ocean?

**Reply by torgemartin** [reply]
... considering the precipitation anomaly could amplify or offset the freshwater forcing we want to prescribe to account for Antarctic/Greenland melting.

**Reply by therealpaulholland** [reply]
To Andrew: I share your inference, if we believe the UKESM projections.  However we expect that UKESM underestimates the dynamic part of the ice loss, since it is not accurately resolving ocean conditions in the Amundsen Sea.  So maybe UKESM is erring on the side of mass gain from SMB.  To Torge: I agree that the models will produce this 'erroneous' FW increase.  It is the difference between this and reality that we need to impose as a forcing, somehow

**Reply by tamsinedwards** [reply]
We do see mass gain this century in many standalone ice sheet simulations too though - if they're forced with climate models with higher accumulation, and also use ice sheet model structures/inputs that favour lower dynamic loss - it's not just UKESM.

**Comment by swartn** [comment]
RE ISMIP6 results for Helene / Sophie. 

In Helene's talk a result is shown that the melt to calving ratio changes significantly into the 21st century. This could be very important. However, if I understand correctly, this was the result from a single ISM. In some other cases (for basal melt), I believe the results were shown for multiple ISMs, but using forcing from only one GCM.

My question is about uncertainty. If looking across multiple ISMs (and preferably multiple GCM driving), how robust is the change in calving to melt ratio? Similarly, how much more uncertain would the basal melt changes be, if multiple GCM drivings had been used? I think understanding the scope of these uncertainties is key if such forcing were to be applied as freshwater by GCMs in e.g. in ScenarioMIP (and indeed, is of interest to us in SOFIA). Thanks!

**Reply by hseroussi** [reply]
Indeed, most of the results I showed were from the entire ensemble, except for the calving fluxes. This is something that is not always reported in the models as it can be tricky to calculated and that has not really been explored as part of ISMIP6, so this results was based on a single model chosen quite randomly. I don't know how the ratio would evolve for the other models, but I would expect a similar behavior with a reduction of the calving flux as ice shelves experience a large amount of thinning. This is something that can be explored further from the other results if needed.

**Reply by therealpaulholland** [reply]
Are we in a position to say anything, do you think?  It all comes down to the calving laws. We won't have large, thin ice shelves in the future if they calve off instead.  It feels to me like this is a question with huge structural uncertainty in the ice modelling.

**Reply by nicojourdain** [reply]
The remote sensing estimates, like [Davison et al. (2023)](https://doi.org/10.1126/sciadv.adi0186)'s Fig. 1, show the complexity. In the Amundsen and Bellingshausen Seas, ice-shelf melting is generally higher than calving, but the partial disintegration of Thwaites made it close to half-half overall in the Amundsen sector (in [Rignot 2013](https://doi.org/10.1126/science.1235798)'s estimate before the disintegration and for steady calving, melting  was slightly more important than calving). So it is possible that the ice-shelf melt fraction gets higher in warmer conditions, but this may reverse at some point. Still, the ISMIP6 numbers may be a good indication and if you go in the details, many of the calved icebergs from Thwaites are still there, so it would not be too bad to count their melting as ice-shelf melting in a simulation.

**Comment by mankoff** [comment]
On 2024-02-14 at 04:21 +13, Neil Swart ***@***.***> wrote...
> In Helene's talk a result is shown that the melt to calving ratio
> changes significantly into the 21st century. This could be very
> important. However, if I understand correctly, this was the result
> from a single ISM. In some other cases (for basal melt), I believe the
> results were shown for multiple ISMs, but using forcing from only one
> GCM.

I note that the term `basal melt` is used above (and elsewhere by @crodehacke). In general I think it is clear from context what is meant by this (i.e., ice shelf basal melt as used here). But I want to point out that to many of us `basal melt` includes or is only from `grounded ice`. I don't think this has been discussed yet, perhaps because it is a small contributor.

In Greenland, grounded ice basal melt is ~5 % of total freshwater discharge (http://doi.org/10.1038/s41467-021-23739-z @nbkglaciology). I don't think this has been constrained in Antarctica, but think it is closer to 10 %.

Basal melt (grounded) in both Greenland and Antarctica is roughly steady state in time (increasing slightly with increased surface runoff in Greenland, and increased basal velocity under both ice sheets). It may therefore be a larger relative contributor earlier in the record.

It is also roughly spatially uniform (due to geothermal), which I speculate means it is a larger % contributor in N. Greenland to the Arctic ocean because there is less calving and SMB runoff there. This touches on @jlbamber comment (https://github.com/NASA-GISS/freshwater-forcing-workshop/discussions/6#discussioncomment-8455821) about FW to the Arctic ocean.


I tried to define these terms and estimate rough % in my 2-slide presentation, somewhere in the Observations folder on the Google Drive.

Finally, since I've opened the discussion here to non-SMB and non-shelf-melt terms (everything else admittedly a smaller magnitude to those two), I want to point out that ground runoff in Greenland (rain & melted snow off the ice sheet) is a non-trivial source. I estimate it at ~30 % of the ice sheet SMB 'runoff' term, or ~15 % of total freshwater from the ice sheet.

**Reply by crodehacke** [reply]
I'm sorry for not being clear enough, but I was talking about "the basal melting in ice shelves"; hence, I have spoken about the basal melting of floating ice shelves, not grounded ice. I'll update my text above .... ;-)

**Reply by mankoff** [reply]
No need to apologize. It was clear! I was just using this as a launch point to bring in other freshwater sources, even if they are smaller contributors.

**Comment by xylar** [comment]
@gavinny, in our discussion yesterday it came up that we wouldn't want a negative total freshwater flux (i.e. "normal" ESM flux plus prescribed anomaly).  I do think there's a danger of this happening under future forcing if we're not careful.  Suppose we create an anomaly dataset (e.g. from ISMIP6 SSP-585 results) that shows a net mass gain for Antarctica (e.g. because ice-shelf basal melting has not yet increased very much but precipitation has).  Suppose I then use that in an ESM that adds that negative anomaly to a mass-balance freshwater flux (you called [a "type 2" model ](https://github.com/NASA-GISS/freshwater-forcing-workshop/discussions/5#discussioncomment-8457390)).  If my ESM has much lower precipitation than the ESM that was used to force the ISMIP6 models, it would be easy for my mass-balance freshwater flux to be smaller than the negative freshwater flux anomaly, leading to a total negative freshwater flux.
